package.json

```diff
@@ -51,6 +51,6 @@
     "@types/react": "^18.3.13",
     "cypress": "^13.17.0",
     "tailwindcss": "^3.4.16",
-    "typescript": "^4.9.5"
+    "typescript": "^5.7.2"
```

Comment:
We've discussed that the latest version of TS is causing issues. We should have TS 4.9.5 and not 5.7.2